### ISSUE TITLE
[BUG] false positive on arbitrary value with space

### ISSUE THREAD

**Issue by jamesarosen** [francoismassart/eslint-plugin-tailwindcss#188]
**Describe the bug**
`no-custom-classname` triggers on an arbitrary-value classname where the value has a space. This happens because the parser splits on space before taking `[…]` into account.

**To Reproduce**
Works:
```html
<div className="tw-grid tw-grid-cols-[1fr]>
```

Doesn't work:
Works:
```html
<div className="tw-grid tw-grid-cols-[1fr 1fr]>
```

**Expected behavior**
I should be able to include spaces within the `[…]` value.

**Screenshots**
![broken-with-space](https://user-images.githubusercontent.com/2406/206578975-c521c06c-d916-4e58-8219-e4e6f9f487b5.png)

**Environment (please complete the following information):**
 - eslint@8.24.0
 - eslint-plugin-tailwindcss@3.7.1

**Comment by francoismassart** [comment]
Hi @jamesarosen,
Thank you for reporting the issue. 

I believe this not a bug because whitespaces are used to separate classnames in the class attribute’s value of HTML. 

You should replace them by underscores like they explain in the official documentation 
See https://tailwindcss.com/docs/adding-custom-styles#handling-whitespace 

Please try that and if it fixes your problem then close this issue. 

Cheers 😀
